package.json

```diff
@@ -74,15 +74,21 @@
+		"makepot:android:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --include=*.native.js,*.android.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./layout-grid.pot --ignore-domain layout-grid-android.pot",
+		"makepot:ios:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./gutenberg.pot --ignore-domain gutenberg-ios.pot",
+		"makepot:ios:jetpack": "npm run wp -- i18n make-pot ./jetpack/projects/plugins/jetpack/extensions --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./jetpack.pot --ignore-domain jetpack-ios.pot",
+		"makepot:ios:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --include=*.native.js,*.ios.js --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/* --skip-php --subtract=./layout-grid.pot --ignore-domain layout-grid-ios.pot",
 		"premakepot:gutenberg": "npm run clean:gutenberg && npm run build:gutenberg",
 		"makepot:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/packages --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --ignore-domain gutenberg.pot",
 		"postmakepot:gutenberg": "npm run clean:gutenberg",
-		"pregenstrings": "npm run makepot:gutenberg",
+		"makepot:jetpack": "npm run wp -- i18n make-pot ./jetpack/projects/plugins/jetpack/extensions --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --ignore-domain jetpack.pot",
+		"makepot:layout-grid": "npm run wp -- i18n make-pot ./block-experiments/blocks/layout-grid --exclude=test/*,e2e-tests/*,build/*,build-module/*,build-style/*,*.native.js,*.ios.js,*.android.js,bundle/* --ignore-domain layout-grid.pot",
+		"pregenstrings": "npm run makepot:gutenberg && npm run makepot:jetpack && npm run makepot:layout-grid",
 		"genstrings": "npm run genstrings:android && npm run genstrings:ios",
-		"genstrings:android": "npm run makepot:android && ./bin/po2android.js gutenberg-android.pot bundle/android/strings.xml",
-		"genstrings:ios": "npm run makepot:ios && ./bin/po2swift.js gutenberg-ios.pot bundle/ios/GutenbergNativeTranslations.swift",
+		"genstrings:android": "npm run makepot:android:gutenberg && npm run makepot:android:jetpack && npm run makepot:android:layout-grid && ./bin/po2android.js bundle/android/strings.xml gutenberg-android.pot jetpack-android.pot layout-grid-android.pot",
+		"genstrings:ios": "npm run makepot:ios:gutenberg && npm run makepot:ios:jetpack && npm run makepot:ios:layout-grid && ./bin/po2swift.js bundle/ios/GutenbergNativeTranslations.swift gutenberg-ios.pot jetpack-ios.pot layout-grid-ios.pot",
```

Comment:
As we do for Gutenberg, we have also to include the POT files generated for each plugin (Jetpack + Layout Grid) when generating the localization strings files.